doc/source/whatsnew/v0.18.0.txt

```diff
@@ -86,13 +101,78 @@ In addition, ``.round()`` will be available thru the ``.dt`` accessor of ``Serie
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. _whatsnew_0180.api_breaking.extract:
+
+Changes to str.extract
+^^^^^^^^^^^^^^^^^^^^^^
+
+The ``.str.extract`` method takes a regular expression with capture
+groups, finds the first match in each subject string, and returns the
+contents of the capture groups.
 
+**PRIOR** to 0.18.0, ``extract`` could return a ``Series``, ``Index``,
+or ``DataFrame``, depending on the subject and regular expression
+pattern. We concluded that this is confusing from a user perspective,
+so ``extract`` now always returns a ``DataFrame`` (:issue:`11386`).
 
+Previously, calling ``series.str.extract`` on a regex with exactly one
+capture group returned a ``Series``:
 
+.. code-block:: python
+
+   In [3]: s = pd.Series(["a1", "b2", "c3"], ["A11", "B22", "C33"])
+   In [10]: one_group = "(?P<letter>[a-zA-Z])"
+   In [11]: s.str.extract(one_group)
+   Out[11]: 
+   A11    a
+   B22    b
+   C33    c
+   Name: letter, dtype: object
+    
+The new behavior is to return a ``DataFrame`` with one column:
+
+.. ipython:: python
 
+   s = pd.Series(["a1", "b2", "c3"], ["A11", "B22", "C33"])
+   one_group = "(?P<letter>[a-zA-Z])"
+   s.str.extract(one_group)
 
+Previously, calling ``index.str.extract`` on a regex with exactly one
+capture group returned an ``Index``:
+
```

Comment:
ok that sound good. just making sure we have coverage of the cases that changed.